### DISCUSSION TITLE
Seeking Clarification about SqlServer on windows-2022

### DISCUSSION THREAD

**Discussion by klreeher** [actions/virtual-environments#9368] [Installed software]
I'd like to know what the authentication mode settings are on SqlServer on the windows-2022 image. 

I'm running into an issue while running my pipeline that appears to be possibly due to the authentication method for sqlserver being set to windows auth only. So I'd like to check if that *is* what the sql server on the windows-2022 image is actually set to be by default.

**Comment by v-sergei-pyshnoi** [comment]
Hello @klreeher . During image generation we do not specify default auth for mysql. You can see full installation script for it [here](https://github.com/actions/runner-images/blob/main/images/windows/scripts/build/Install-MysqlCli.ps1).
